test-data/add.yaml

```diff
@@ -175,3 +175,58 @@ post:
   - r7.packet_offset-r7.uvalue<=-7
   - r7.svalue-r7.packet_offset<=11
   - r7.uvalue-r7.packet_offset<=11
+
+---
+test-case: check concrete state
+
+pre: ["r0.type=number", "r0.svalue=0", "r0.uvalue=0", r1.type=number, r1.svalue=0, r1.uvalue=0, r2.type=number, r2.svalue=0, r2.uvalue=0]
+
+code:
+  <start>: |
+    r1 = 1
+    require [r1.type=number, r1.svalue=1, r1.uvalue=r1.svalue]
```

Comment:
I believe the intention is to allow these numbers, not require, so maybe my suggestion was not the right terminology. Maybe it should be "require allow ..."? There must be a better term.